pkg/network/pod/annotations/generator_test.go

```diff
@@ -445,28 +445,28 @@ var _ = Describe("Annotations Generator", func() {
 
 			expectedNetInfo := []downwardapi.Interface{
 				{
-					Network: networkName2,
+					Network: networkName3,
```

Comment:
> Following PR https://github.com/kubevirt/kubevirt/pull/14477, the JSON array within the kubevirt.io/network-info annotation's value, is expected to be ordered by the network names.

what do i miss, how come networkName3 comes before networkName2 ?
unless some internal processing is done, and this reflects just the end result but not the json

Reply:
> > Following PR #14477, the JSON array within the kubevirt.io/network-info annotation's value, is expected to be ordered by the network names.
> 
> what do i miss, how come networkName3 comes before networkName2 ? unless some internal processing is done, and this reflects just the end result but not the json

The order is lexicographic:
```go
networkName2 = "foo"
networkName3 = "doo"
```
That is why, in this case, net3 comes before net2.

Reply:
ah, thanks
thought for some reason networkName2 = "networkName2",

well, might worth just because of this reason to change the var name + value to meet the order in both ?

Reply:
In this specific test, the intention is to show that the output JSON array elements are ordered by network name in lexicographical order, even if the order of spec interfaces / networks / multus-network-status elements are ordered differently.

Reply:
of course, but i personally dont like the names foo / doo and their reverse order here, 
the fact their var name is reversed adds another layer of thinking that otherwise wont be required,
especially when here we care about the ordering